recipe/meta.yaml

```diff
@@ -30,15 +34,27 @@ test:
     - hatch_jupyter_builder
   commands:
     - pip check
-    - pip install -e ".[test]"
-    - python -m pytest -vv
+# Note windows tests supressed because of problems with utils.run(shlex.split(cmd))
+#    - python -m pytest -k "not (test_run or test_normalize_cmd)" -vv  # [win]
+    - pip install -e ".[test]" # [not (win or s390x)]
```

Comment:
You specified the dependencies in the `requires` section so you don't need this. Also the package will be already installed (by conda-build), so no need to install.